community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/resume.py

```diff
@@ -459,13 +459,14 @@ def hold_job(job_id, reason):
     run(f"{lookup().scontrol} update jobid={job_id} comment='{reason}'")
 
 
-def create_placement_request(pg_name: str, region: str, max_distance: Optional[int]):
+def create_placement_request(pg_name: str, region: str, max_distance: Optional[int], gpu_topology: Optional[str]):
```

Comment:
nit. rename `gpu_topology` -> `accelerator_topology` where applicable, to make codebase less confusing and more "searchable".